Calendars/Calendars.Plugin.MAUI/Calendars.Plugin.MAUI.csproj

```
@@ -0,0 +1,63 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <TargetFrameworks>net6.0;net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
+      <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows')) and '$(MSBuildRuntimeType)' == 'Full'">$(TargetFrameworks);net6.0-windows10.0.19041</TargetFrameworks>
+      <UseMaui>true</UseMaui>
+      <SingleProject>true</SingleProject>
+      <ImplicitUsings>enable</ImplicitUsings>
+      <GeneratePackageOnBuild>False</GeneratePackageOnBuild>
+      <GenerateDocumentationFile>True</GenerateDocumentationFile>
+
+      <SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-ios'">14.2</SupportedOSPlatformVersion>
+      <SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-maccatalyst'">14.0</SupportedOSPlatformVersion>
+      <SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-android'">21.0</SupportedOSPlatformVersion>
+      <SupportedOSPlatformVersion Condition="$(TargetFramework.Contains('-windows'))">10.0.17763.0</SupportedOSPlatformVersion>
+      <TargetPlatformMinVersion Condition="$(TargetFramework.Contains('-windows'))">10.0.17763.0</TargetPlatformMinVersion>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <Compile Include="..\Calendars.Plugin.Abstractions\Calendar.cs" Link="Calendar.cs" />
+      <Compile Include="..\Calendars.Plugin.Abstractions\CalendarEvent.cs" Link="CalendarEvent.cs" />
+      <Compile Include="..\Calendars.Plugin.Abstractions\CalendarEventReminder.cs" Link="CalendarEventReminder.cs" />
+      <Compile Include="..\Calendars.Plugin.Abstractions\ICalendars.cs" Link="ICalendars.cs" />
+      <Compile Include="..\Calendars.Plugin.Abstractions\ICalendarsExtensions.cs" Link="ICalendarsExtensions.cs" />
+      <Compile Include="..\Calendars.Plugin.Abstractions\PlatformException.cs" Link="PlatformException.cs" />
+
+      <Compile Include="..\Calendars.Plugin\CrossCalendars.cs" Link="CrossCalendars.cs" />
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(TargetFramework)' == 'net6.0-android'">
```

nit (and similar one for others): 
```suggestion
    <ItemGroup Condition="$(TargetFramework.Contains(-android))">
```

---

😅, thanks - I actually did it that way in my other projects 🙈